app/src/main.js

```diff
@@ -8,25 +7,18 @@ import App from './App.vue'
 import './registerServiceWorker'
 import store from './store'
 import router from './router'
+import apollo from './modules/gql/apolloClient'
 
 const BASE = window.location.origin
 const uri = `${BASE}/api/graphql`
 
-const apolloClient = new ApolloClient({
-  uri
-})
-
-const apolloProvider = new VueApollo({
```

Comment:
We still need this line. We should be calling that apollo function here

Reply:
It's also in line 23 of the other file. Should I take it out there?

Reply:
In your originally suggestion for how to write this function, you sent this: 
```export default const apollo (uri) => {
  const apolloClient = new ApolloClient({
    uri, token: localStorage.getItem('token') || null
  })
  return new VueApollo({
    defaultClient: apolloClient
  })
}```
which has the `new VueApollo` call in it already, so I took it out of main.js

Reply:
I put this back in main and called apollo function from there